### PR TITLE
QoL Change: Last Names List Updated

### PR DESCRIPTION
**Pull request by t-toasted**
## About the PR 
Updates the list of last names to be in alphabetical order.  Deletes duplicate last names. Adds in about 200 more last names that can be found ingame; Arabic, Chinese, Indian, Peruvian, and others. 

Example last names: Xiao, Padmanabhan, Halabi 

## Why's this needed? 
Alphabetical order is good. Also, more variety in last names is even better. 

## Changelog

```changelog
(u)Roddy
(+)Added additional last names to the random name pool.
```

### CODE REVIEW

**Comment by t-toasted** [comment]
Was originally going to be the entire names lists updated with alphabetical/more names added but. Can't make it omega sized or I cry and blow up

File: strings/names/last.txt [hunk 1]
```diff
@@ -1,851 +1,1064 @@
...
+	Tang
+	Tanner
+	Tannous
+	Tao
+	Taylor
+	Teagarden
+	Tedrow
+	Tennant
+	Thomas
+	Thomas
```
Comment (ZeWaka) [inline strings/names/last.txt:937]: ?

Comment (t-toasted) [reply strings/names/last.txt:937]: missed a thomas! Thanks
I went through a few times but seems I missed this one

File: strings/names/last.txt [hunk 2]
```diff
@@ -1,851 +1,1063 @@
...
+	Zhu
+	Zimmer
+	Zoucks
+	de Leon
+	the Eighth
+	the Fourth
+	the Third
+	the Twentieth
+	the Eleventh
+	the French
```
Comment (flrsh) [inline strings/names/last.txt:1063]: Players can use these names themselves.

File: strings/names/last.txt [hunk 3]
```diff
@@ -1,851 +1,1063 @@
...
+	Shupe
+	Sideiron
+	Sidower
+	Siegrist
+	Silver
+	Simmons
+	Simpson
+	Simsek
+	Singh
+	Sitting Bull
```
Comment (flrsh) [inline strings/names/last.txt:871]: It seems in poor taste to just casually add a couple names from some well-known Native Americans, just for the sake of diversifying our name pool. Especially as (1) some of these names were given based on the specific individual (e.g. Sitting Bull) (2) some of these names seem to be questionable translations (e.g. Crow Dog). It just sits weird with me. Would appreciate thoughts from those who are Native American/Indigenous.

**Review by flrsh — COMMENTED** [review]
Did you get these new last names from a particular source / set of sources? How much research did you do?

File: strings/names/last.txt (hunk 2 above)
Comment (t-toasted) [reply strings/names/last.txt:1063]: Can remove these names from the pool. Only one I added here was within the joke category of, 'the French'.

File: strings/names/last.txt [hunk 4]
```diff
@@ -1,851 +1,1063 @@
...
-	Batten
+	Cavalet
+	Celik
+	Cham
+	Chandra
+	Chang
+	Chapman
+	Chasing Bear
+	Chasing Horse
+	Chauvin
```
[the remark this answers is not in the document]
Comment (t-toasted) [reply strings/names/last.txt:173]: Yep!

File: strings/names/last.txt (hunk 3 above)
Comment (t-toasted) [reply strings/names/last.txt:871]: Names drawn from [https://www.marquette.edu/library/archives/Mss/BCIM/BCIM-SC1-NAsurnames.pdf], but also would appreciate this if possible.

Comment (flrsh) [reply strings/names/last.txt:870]: Thanks!

Comment (t-toasted) [reply strings/names/last.txt:870]: > Thanks!

If you'd like, I can remove them and wait until we get feedback to see if it should be added? Only about... 5 or so names to remove, so it wouldn't be much to remove if this is the way you'd prefer!

Comment (flrsh) [reply strings/names/last.txt:870]: Yeah, it just feels weird to me for us / the game to be giving random players these names. That sounds good!

Comment (t-toasted) [reply strings/names/last.txt:870]: > Yeah, it just feels weird to me for us / the game to be giving random players these names without context / history. That sounds good!

Sure thing! I understand completely

**Review by flrsh — APPROVED** [review]
Skimmed the list and it looks fine to me!

File: strings/names/last.txt [hunk 5]
```diff
@@ -1,851 +1,1048 @@
...
+	Mccloud
+	Mcclymonds
+	Mccullough
+	Mccune
+	Mcdonald
+	Mcfall
+	Mcintosh
+	Mckendrick
+	Mcloskey
+	Mcmullen
```
Comment (ZeWaka) [inline strings/names/last.txt:588]: if you're going to have McNames, spell them right
```suggestion
	McCloud
	McClymonds
	McCullough
	McCune
	McDonald
	McFall
	McIntosh
	McKendrick
	McCloskey
	McMullen
```

File: strings/names/last.txt [hunk 6]
```diff
@@ -1,851 +1,1048 @@
...
+	Lowstetter
+	Lucy
+	Ludwig
+	Lugassi
+	Luna
+	Luo
+	Lynch
+	Lysenko
+	Maalouf
+	Maclagan
```
Comment (ZeWaka) [inline strings/names/last.txt:551]:
```suggestion
	MacLagan
```
ditto

File: strings/names/last.txt (hunk 5 above)
Comment (t-toasted) [reply strings/names/last.txt:588]: Those were present in the original name file, I just ordered them. But I can fix them to be proper

**Pull request merged**